webrender/src/resource_cache.rs

```diff
@@ -216,6 +216,20 @@ impl ResourceCache {
         self.texture_cache.add_raw_update(texture_id, size);
     }
 
+    pub fn update_webgl_texture(&mut self, id: WebGLContextId, texture_id: TextureId, size: Size2D<i32>) {
+        { 
+            let prev_texture_id = self.webgl_textures.get(&id).unwrap();
+
+            // Remove existing cache if texture id has changed
+            if !prev_texture_id.eq(&texture_id) {
```

Comment:
any reason `!eq()` and `clone()` are used instead of `!=` and `*` (since this is `Copy`)?